packages/starlight/components/Search.astro

```diff
@@ -43,9 +43,8 @@ const pagefindTranslations = {
       </button>
       {
         import.meta.env.DEV ? (
-          <div style="margin: auto; text-align: center;" dir="ltr">
-            <p>Search is only available in production builds.</p>
-            <p>Try building and previewing the site to test it out locally.</p>
+          <div style="margin: auto; text-align: center; white-space: pre-line;" dir="ltr">
```

Comment:
Given this is now using a translated string, maybe we should remove the `dir="ltr"`? It’s a actually a tricky case I haven’t really accounted for: no way to know the direction of the string that `t()` returns, e.g. on an RTL page, if it’s LTR fallback content or an RTL translation.

Here’s what was happening on RTL pages prior to adding `dir`:

![Search modal showing English text with punctuation at the start of the sentences instead of at the end as expected](https://github.com/withastro/starlight/assets/357379/715fe98a-7ec3-4a9e-a215-3b37acc9dd01)

Not sure. I guess it might be safest to keep it for now given we don’t have any RTL translations of the warning yet.

Reply:
Yeah, I guess it's safest to keep it for now. Wondering if we should build a function like `isFallbackTranslation("search.devWarning")` to handle these cases?